llvm/tools/spirv-to-ir-wrapper/spirv-to-ir-wrapper.cpp

```diff
@@ -46,6 +47,11 @@ static cl::opt<std::string>
     LlvmSpirvOpts("llvm-spirv-opts", cl::value_desc("llvm-spirv options"),
                   cl::desc("options to pass to llvm-spirv"));
 
+// SkipNonLLVMIR - Skip non-LLVM-IR files (create empty output instead)
+static cl::opt<bool>
+    SkipNonLLVMIR("skip-non-llvmir",
```

Comment:
It may be not important, but at first when I read 'skip-non-llvmir' name I thought that this option turns off processing of non-IR files leaving them as is. However, those files are cleared.
I suppose that it may be not simple to make short and clear name for this option. What do you think about 'clear-non-llvmir'?

Reply:
How about `-skip-unknown-inputs`?